### DISCUSSION TITLE
Reading PRM file

### DISCUSSION THREAD

**Discussion by danefreitas** [coin-or/Osi#199] [Ideas]
I'm a user of Osi and would like to know if you have considered the possibility of implementing the reading of parameter files in the .prm format, similar to what is done in solvers like Gurobi and CPLEX. I believe that this functionality could facilitate and standardize parameter configuration.

**Comment by jjhforrest** [comment]
In Cbc/master there is a command -optionfile xxx.xxx which has same format as .prm file.  It uses * to signal a comment.  I will add # as an alternative way to signal a comment.
